modules/rostests/apitests/win32u/CMakeLists.txt

```
@@ -2,4 +2,4 @@
 add_subdirectory(win32u_ros)
 add_subdirectory(win32u_xpsp2)
 add_subdirectory(win32u_2k3sp2)
-#add_subdirectory(win32u_vista)
+add_subdirectory(win32u_vista)
```

Additional suggestion:
```suggestion
add_subdirectory(win32u_vistasp2)
```